python/tests/test_feature_group.py

```diff
@@ -342,8 +344,8 @@ def test_constructor_with_list_event_time_for_compatibility(
     def test_select_all(self):
         query = test_feature_group.select_all()
         features = query.features
-        assert len(features) == 4
-        assert set([f.name for f in features]) == {"pk", "ts", "f1", "f2"}
+        assert len(features) == 5
+        assert set([f.name for f in features]) == {"pk", "fk", "ts", "f1", "f2"}
 
     def test_select_all_exclude_pk(self):
```

Comment:
Shouldn't test `test_select_all_exclude_pk` and `test_select_all_exclude_ts` have "fk" in the assert statement.